docs/source/general/enforcing_limits.rst

```diff
@@ -3,14 +3,30 @@ Enforcing session and bandwidth limits
 ======================================
 
 This feature lets the system administrator enforce session and bandwidth limits on the users. Some of the features added here are,
-* Put daily and all time session limits.
-* Put daily bandwidth limits.
-* Default profile for the new users being added.
+
+    - Put daily and all time session limits.
+    - Put daily bandwidth limits.
+    - Default profile for the new users being added.
 
 Adding from admin inteface
 --------------------------
 
-At the url `/admin/django_freeradius/radiusprofile/` one can add the profiles for different types of users. And add the user profile mappings at the url `/admin/django_freeradius/radiususerprofile/`. This helps admins configure user limits easily.
+At the url ``/admin/django_freeradius/radiusprofile/`` one can add the profiles for different types of users. And add the user profile mappings at the url ``/admin/django_freeradius/radiususerprofile/``. This helps admins configure user limits easily. There are attributes defined in the RadiusProfile model which help us enforce limits on the users.
```

Comment:
Was trying to test this in the master branch.  I don't see a url mapping for /admin/django_freeradius/radiususerprofile/
```
^admin/ auth/group/
^admin/ django_freeradius/radiuscheck/
^admin/ django_freeradius/radiusreply/
^admin/ django_freeradius/radiusaccounting/
^admin/ django_freeradius/nas/
^admin/ django_freeradius/radiususergroup/
^admin/ django_freeradius/radiusgroupreply/
^admin/ django_freeradius/radiusgroupcheck/
^admin/ django_freeradius/radiuspostauth/
^admin/ django_freeradius/radiusbatch/
^admin/ django_freeradius/radiusprofile/
```

Reply:
@2stacks it's managed directly in the user page

Reply:
Hey @2stacks thanks for pointing out. Now it has been changed.